app/budget/directives/courseCosts/courseCostRow/courseCostRow.css

```diff
@@ -20,4 +20,48 @@
 
 .course-cost-row .show-warning {
 	visibility: visible;
+}
+
+.course-cost-row input {
+	/* bound copy: */
+background: #FFFFFF;
+border: 1px solid #CCCCCC;
+box-shadow: inset 0 1px 2px 0 rgba(131,131,131,0.30);
+width: 50%;
+padding: 5px;
+border-radius: 3px;
+text-align: center;
+}
+
+
+
+
+/* test */
+
+.Rtable {
+    display: -webkit-box;
```

Comment:
All styles in this file should be prefixed with .course-cost-row to avoid them potentially affecting different components that use the same class names/selectors

Example:
.Rtable {
Should instead be:
.course-cost-row .Rtable {

Reply:
These Rtable classes have an odd naming structure. I'm guessing this is demo code  you found online? :)

As you work through the remains of the old budgetView CSS, try to create new classes (and rewrite/delete old ones) towards BEM conventions. When we're both in the office again I can walk through it in more detail with you if you have questions.
http://getbem.com/naming/

Also, it doesn't look like this selector matches anything in the html (unless I'm missing something) - are these orphaned classes from something you were experimenting with? You should always delete CSS when it no longer is needed.